src/main/java/codewars/tasks/java/Main.java

```diff
@@ -24,6 +25,8 @@ public static void main (String [] args){
         Battle.declareWinner(new Fighter("Lew", 10, 2),
                 new Fighter("Harry", 5, 4), "Lew");
 
+        TemperatureAnalysis analysis = new TemperatureAnalysis();
```

Comment:
Please create action package where will be separate classes with main method. 
ForExample LeapYearExecution.class, TemperatureAnalysis.class